test/Dialect/FIRRTL/SFCTests/mem-taps.fir

```diff
@@ -1,5 +1,6 @@
 ; RUN: firtool --verilog %s | FileCheck %s
 ; RUN: firtool --verilog -preserve-aggregate=1d-vec %s | FileCheck %s --check-prefix=AGGGREGATE
+; RUN: firtool --verilog -lower-annotations-no-ref-type-ports %s | FileCheck %s --check-prefix=NOREFS
```

Comment:
Please also test this behavior on the `mem-taps-reg.fir` example to ensure that works and stays working.

(not sure about where else might be appropriate to check this works/interacts in the right way?)